stable/busybox/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for busybox.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: busybox
+  tag: latest
+  pullPolicy: IfNotPresent
+service:
+  name: busybox
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 80
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
I think it is best practice not to set resources by default. Here's an example: https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/values.yaml#L41